stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md

```diff
@@ -24,14 +24,14 @@ endpoint jdbc:Client testDB {
     dbOptions: { useSSL: false }
 };
 ```
-The full list of endpoint properties can be found in the `sql:PoolOptions` type.
+The full list of endpoint properties can be found listed under the `sql:PoolOptions` type located in the `types.bal` file of the SQL package directory.
 
 ### Creating tables
 
-Following is an example of creating a table with two columns. One field is of type int and the other of varchar. The CREATE statement is executed via the `update` operation of the endpoint.
+This sample creates a table with two columns. One field is of type `int` and the other of is of type `varchar`. The CREATE statement is executed via the `update` operation of the endpoint.
```

Comment:
Comma after `int`?

Reply:
Yes, insert comma after `int`

Reply:
Fixed with this commit: https://github.com/ballerina-platform/ballerina-lang/pull/8071/commits/b100133f6879d70445f59461cd7a1c84ecc1c05b